modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIApplication.macro

```diff
@@ -0,0 +1,138 @@
+definition {
```

Comment:
Maybe it would be better to break the macro into two files, UI and API/JSON methods?